tools/dependencies/known-oap-backend-dependencies.txt

```diff
@@ -145,22 +150,17 @@ protobuf-java-util-3.11.4.jar
 rank-eval-client-6.3.2.jar
 reactive-streams-1.0.2.jar
 reflectasm-1.11.7.jar
-resourcecify-annotations-0.9.2.jar
+resourcecify-annotations-0.21.0.jar
 retrofit-2.3.0.jar
 simpleclient-0.6.0.jar
 simpleclient_common-0.6.0.jar
 simpleclient_hotspot-0.6.0.jar
 simpleclient_httpserver-0.6.0.jar
 slf4j-api-1.7.25.jar
 snakeyaml-1.18.jar
-sundr-codegen-0.9.2.jar
-sundr-core-0.9.2.jar
-swagger-annotations-1.5.12.jar
+sundr-codegen-0.21.0.jar
+sundr-core-0.21.0.jar
+swagger-annotations-1.5.22.jar
```

Comment:
You should update all these changes(version and new libs) into the LICENSE file. Also, if any of new lib is in Apache 2.0 license, and having NOTICE file in their original repo, you need to copy the content into our NOTICE file too(in the same folder as LICENSE file)

All these requirements are based on the license laws and ASF guidances.